### PR TITLE
Reduce log outputs from error states.

### PR DESCRIPTION
**Pull request by mikepurvis**
Policy is to error once/infrequently; debug each time it occurs.

Resolves CORE-2881, CORE-2882, CORE-2883, CORE-2884, CORE-2892.

@abencz @safrimus

### CODE REVIEW

**Comment by safrimus** [comment]
LGTM

File: src/sicks3000.cc [hunk 1]
```diff
@@ -81,7 +81,8 @@ int SickS3000::Open()
   // Setup serial device
   if (this->serial->OpenPort2() == SERIAL_ERROR)
   {
-    ROS_ERROR("SickS3000::Open: Error Opening Serial Port");
+    ROS_ERROR_ONCE("SickS3000::Open: Error Opening Serial Port");
```
Comment (abencz) [inline src/sicks3000.cc:84]: ONCE means this won't show up multiple times if the cable is reconnected multiple times?

**Comment by abencz** [comment]
LGTM.

File: src/sicks3000.cc (hunk 1 above)
Comment (mikepurvis) [inline src/sicks3000.cc:84]: The theory is that an error which prevents the port from opening is probably permanent.

Comment (abencz) [inline src/sicks3000.cc:84]: Wouldn't the cable being unplugged be non-permanent?

Comment (mikepurvis) [inline src/sicks3000.cc:84]: Maybe on someone's desk, probably not buried in a robot.

**Pull request merged**